practice_independent_challenges/week3/world_cup/spec/player_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/player'
+
+RSpec.describe Player do
+  it 'exists' do
+    player = Player.new({name: "Luka Modric", position: "midfielder"})    
+
+    expect(player).to eq (player)
```

Comment:
For this test, you want to expect that `player` is an instance of the Player class. So something like:
```
expect(player).to be_instance_of(Player)
```